.github/workflows/extensions-test.yml

```diff
@@ -76,4 +56,4 @@ jobs:
       - name: test
         run: |
           cd libsql-sqlite3/test/rust_suite
-          cargo test --features extensions
\ No newline at end of file
+          cargo test --features extensions
```

Comment:
this part targets all tests, including the one in extensions module. 

What we want is, just run the extensions module tests, but nothing else.

Reply:
Actually, I do not familiar with the Rust and Rust test command, just trying to work on the CI side but it seems complicated than I thought. So was not able to figure out the right `cargo test` command to run the extensions module tests only so far. If you have any hint or document related to this, please let me know. Thanks!

Reply:
docs: https://doc.rust-lang.org/cargo/commands/cargo-test.html#target-selection

something like this would work:

```
cargo test --features extensions --test tests extensions
```

Reply:
Thanks, I will check it to find the appropriate command for this case.

Reply:
Finally, I've found the command to run only extension test, it works on my forked repo
https://github.com/tungbq/libsql/actions/runs/8028972291/job/21934748847#step:6:474
![image](https://github.com/tursodatabase/libsql/assets/85242618/32543bec-b9f8-4442-93e4-8371bd3fbf0a)
Hi @avinassh please help review again! Thanks

Reply:
@tungbq looks great! can you squash all the commits into a single one?

Reply:
It's done @avinassh